### ISSUE TITLE
Accessibility notifications aren't firing

### ISSUE THREAD

**Issue by iKenndac** [iKenndac/xcode-streamdeck-plugin#1]
The `XcodeObserver` class should be receiving notifications from the accessibility system, but it isn't when it's running in the plugin (it works in a regular AppKit app). Perhaps it's because the plugin doesn't have a bundle identifier etc.
